### PR TITLE
Pigeon & Odometry Works!

### PR DESCRIPTION
**Pull request by BenWertz**
Pull Request Checklist:
- [x] You are able to understand what the code is doing on a basic level with the use of comments
- [x] Code is formatted in a way that isn't confusing
- [x] builds cleanly
- [x] builds *to the robot* cleanly
- [x] Code has been tested on the robot/roadkill/other mechanical setup and completes the task it needs to complete

### CODE REVIEW

**Review by Linkd-Aether — DISMISSED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -108,6 +109,7 @@ public void registerSubsystem(SubsystemBase subsystem, CommandBase command) {
    */
   @Override
   public void robotPeriodic() {
+    // OI.driverController.setRumble(RumbleType.kLeftRumble, 65536);
```
Comment (jjesse1972) [inline src/main/java/frc/robot/Robot.java:112]: remove these commented lines unless we intend to uncomment soon.

File: src/main/java/frc/robot/commands/DriveControls.java [hunk 2]
```diff
@@ -49,7 +50,7 @@ public void initialize() {
      * @return axisValue with throttle multiplier
      */
     private double addMultiplier(double axisValue, double multiplier_) {
-        return axisValue * (0.25 + multiplier_ * 0.75);
+        return axisValue * (0.125 + multiplier_ * 0.875);
```
Comment (jjesse1972) [inline src/main/java/frc/robot/commands/DriveControls.java:53]: Do we want these changes to go to master?

File: src/main/java/frc/robot/commands/DriveControls.java [hunk 3]
```diff
@@ -103,18 +104,19 @@ private void arcadeCompute() {
     public void execute() {
         multiplier = Utility.deadzone(OI.driverController.getRawAxis(3));
 
-        forward = Utility.deadzone(OI.driverController.getRawAxis(1));
-        rotation = Utility.deadzone(OI.driverController.getRawAxis(4));
+        forward = addMultiplier(deadzone(OI.driverController.getRawAxis(1)));
```
Comment (jjesse1972) [inline src/main/java/frc/robot/commands/DriveControls.java:107]: do we want these changes to go to master

File: src/main/java/frc/robot/commands/DriveControls.java [hunk 4]
```diff
@@ -103,18 +104,19 @@ private void arcadeCompute() {
...
 
         rotation *= -1;
 
         if (drivetrain.isDrivetrainEngaged()) {
             arcadeCompute();
-        
             // passes the final axis values into the drivetrain
-            drivetrain.setPower(limit(addMultiplier(leftOutput)), -limit(addMultiplier(rightOutput)));
+            // drivetrain.setPower(limit(addMultiplier(leftOutput)), -limit(addMultiplier(rightOutput)));
+            drivetrain.setRotationalVelocity(500*limit((leftOutput)), -500*limit((rightOutput)));
```
Comment (jjesse1972) [inline src/main/java/frc/robot/commands/DriveControls.java:116]: remove these commented lines unless we intend to uncomment soon.

File: src/main/java/frc/robot/subsystems/instances/Drivetrain.java [hunk 5]
```diff
@@ -187,7 +202,7 @@ public void setPower(double left, double right) {
      */
     public DifferentialDriveWheelSpeeds getWheelSpeeds() {
         return new DifferentialDriveWheelSpeeds(leftMotorLeader.getSelectedSensorPosition() / ticksPerMeter,
-                rightMotorLeader.getSelectedSensorPosition() / ticksPerMeter);
+                -rightMotorLeader.getSelectedSensorPosition() / ticksPerMeter);
```
Comment (jjesse1972) [inline src/main/java/frc/robot/subsystems/instances/Drivetrain.java:210]: remove these commented lines unless we intend to uncomment soon.

File: src/main/java/frc/robot/subsystems/instances/Drivetrain.java [hunk 6]
```diff
@@ -324,16 +342,22 @@ public void engageDrivetrain() {
 
         leftMotorLeader.setSensorPhase(true);
         rightMotorLeader.setSensorPhase(true);
-
-        double P = 0.05;
-        double I = 0;
+        double P = 1.1e-1;
```
Comment (jjesse1972) [inline src/main/java/frc/robot/subsystems/instances/Drivetrain.java:350]: remove these commented lines unless we intend to uncomment soon.

**Review by jjesse1972 — CHANGES_REQUESTED** [review]

**Review by 122004 — APPROVED** [review]

**Review by Linkd-Aether — APPROVED** [review]

**Review by jjesse1972 — APPROVED** [review]

**Pull request merged**